devops/Makefile

```diff
@@ -285,6 +307,33 @@ create-new-db:
 	@echo "+\n++ Creating DB ($(NEW_DB)) in $(NAMESPACE) patroni db cluster"
 	@oc -n $(NAMESPACE) process -f openshift/database/createdb-job.yml -p DB_NAME=$(NEW_DB) -p BUILD_NAMESPACE=$(BUILD_NAMESPACE) -p JOB_NAME=$(JOB_NAME) | oc -n $(NAMESPACE) apply -f -
 
+run-db-migrations:
+	test -n "$(NAMESPACE)"
+	test -n "$(BUILD_NAMESPACE)"
+	test -n "$(DB_SERVICE_NAME)"
+	test -n "$(DB_MIGRATIONS_BUILD_REF)"
+	test -n "$(BUILD_REF)"
+	test -n "$(DATABASE_NAME_KEY)"
+	test -n "$(DB_MIGRATIONS)"
+	@echo "+\n++ Deploying DB_MIGRATIONS with tag: $(BUILD_REF)\n+"
+	@oc -n $(NAMESPACE) process -f openshift/db-migrations-deploy.yml \
+		-p NAME=$(DB_MIGRATIONS) \
+		-p DB_SERVICE=$(DB_SERVICE_NAME) \
+		-p BUILD_NAMESPACE=$(BUILD_NAMESPACE) \
+		-p IMAGE_STREAM_TAG="$(DB_MIGRATIONS_BUILD_REF):$(BUILD_REF)" \
+		-p BUILD_TAG=$(BUILD_REF) \
+		-p DATABASE_NAME_KEY=$(DATABASE_NAME_KEY) \
+		-p JOB_NAME=$(DB_MIGRATIONS_JOB_NAME) \
+		| oc -n $(NAMESPACE) apply -f - 
+
+wait-for-db-migrations:
```

Comment:
Are we taking this approach using `wait-for-db-migrations` because there is no way to call something **similar** to `$(call rollout_and_wait,dc/$(API))` as API, Workers, etc.?

Reply:
Yes for jobs we have only wait, and check the status. So enhanced a little bit with some max timeout and checking status for every 5 sec. As this is the main job before the other jobs run.

Reply:
Thanks for doing the changes and removing the `wait-for-db-migrations`.